content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md

```diff
@@ -0,0 +1,175 @@
+
+For example:
+
+`$ export LCG_GFAL_INFOSYS=egee-bdii.cnaf.infn.it:2170`
+
+`$ lcg-infosites --vo ops ce | grep nikhef`
+`  5680      15          0            0       0 dissel.nikhef.nl:2119/jobmanager-pbs-infra`
+`  5680      17          1            1       0 gazon.nikhef.nl:8443/cream-pbs-infra`
+`  5680      15          0            0       0 juk.nikhef.nl:8443/cream-pbs-infra`
+`  5680      15          0            0       0 klomp.nikhef.nl:8443/cream-pbs-infra`
+`  5680      16          0            0       0 stremsel.nikhef.nl:8443/cream-pbs-infra`
+
+For getting the GlueServiceEndpoint URL of your SRM service, you can
+launch a ldap query to your site-bdii (or also directly to the SRM
+service):
+
+`$ ldapsearch -x -LLL -H `<ldap://sbdii01.ncg.ingrid.pt:2170>` -b "mds-vo-name=NCG-INGRID-PT,o=grid" '(&(objectClass=GlueService)(GlueServiceType=SRM))' GlueServiceEndpoint`
+
+`dn: GlueServiceUniqueID=httpg://srm01.ncg.ingrid.pt:8444/srm/managerv2,Mds-Vo-name=NCG-INGRID-PT,o=grid`
+`GlueServiceEndpoint: `**`httpg://srm01.ncg.ingrid.pt:8444/srm/managerv2`**
+
+In a similar way, by just changing the value of *GlueServiceType*, you
+can get the endpoint URLs of other services.
+
+An alternative way for retrieving the GlueServiceEndpoint URL is using
+the GLUE2 information browser provided by VAPOR
+(https://operations-portal.egi.eu/vapor/resources/GL2ResNGISites):
+select your NGI, then your site and hence the Storage service; click on
+the *endpoint details* button for finding the URL associated to the SRM
+interface.
+
+## Fill in the information
+
+### URLs information are completely missing
+
+  - URLs information are completely missing: you need to edit the
+    services information
+
+|                                                                                                                                                    |                                                          |
+| -------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| this is the home page regarding your site. You need to fill in the URL information. Click on a service for displaying its page (e.g. the CREAM-CE) | [frameless|border|800px](/File:Picture01.png "wikilink") |
+| click on the EDIT button on the top-right corner                                                                                                   | [frameless|border|800px](/File:Picture02.png "wikilink") |
+| fill in the *Service URL* field with the queue URL;                                                                                                | [frameless|border|800px](/File:Picture03.png "wikilink") |
+| Now the CREAM-CE service endpoint contains the required queue information. Proceed in a similar way for the other CREAM-CE and SRM services.       | [frameless|border|800px](/File:Picture04.png "wikilink") |
```

Comment:
Lots of CREAM-CE that is no longer in use

Reply:
Will leave CREAM until HTCondor or else info is added.